### DISCUSSION TITLE
Keycloak cache configuration with Redis

### DISCUSSION THREAD

**Discussion by vivekpratapsingh** [keycloak/keycloak#24210] [Q&A]
Hi Team, We have a centralized caching mechanism with Redis for other applications. So we want to use Redis as store for Keycloak v21 cache mechanism instead of infinispan. Do we have any documents/instructions to configure the same? Also is it possible to configure the the caches for AuthenticationSessionProvider and UserSessionProvider only and keep the infinispan as store for cache mechanism of other providers?

**Comment by stianst** [comment]
We do not plan to add support to Redis directly, see https://github.com/keycloak/keycloak/discussions/48577 for more details.
